Framework/DataObjects/test/PeaksWorkspaceTest.h

```diff
@@ -485,6 +485,28 @@ class PeaksWorkspaceTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(wsCastConst, wsCastNonConst);
   }
 
+  void test_removePeaks() {
+    // build peaksworkspace (note number of peaks = 1)
+    auto pw = buildPW();
+    Instrument_const_sptr inst = pw->getInstrument();
+
+    // add peaks
+    Peak p(inst, 1, 3.0);
+    Peak p2(inst, 2, 6.0);
+    Peak p3(inst, 3, 9.0);
+    pw->addPeak(p);
+    pw->addPeak(p2);
+    pw->addPeak(p3);
+
+    // number of peaks = 4, now remove 3
+    std::vector<int> badPeaks;
```

Comment:
Use `std::vector<int> badPeaks {0, 2, 3};` so we don't have to do 3 push backs vs one op.